main.py

```diff
@@ -164,10 +173,18 @@ def find_phone(self, phone_number):
     def add_birthday(self, birthday):
         self.birthday = Birthday(birthday)
 
+    def add_adress(self, new_address):
+        self.address = Address(new_address)
+
+    def add_email(self, new_email):
+        self.email = Email(new_email)
+
```

Comment:
I think we can add birthday as it related to contact.
Yes we have independent functionality for this but better add birthday